NavigationReactNative/src/TabBar.tsx

```diff
@@ -18,14 +19,15 @@ class TabBar extends React.Component<any, any> {
         return !!selectedTab;
     }
     render() {
-        var {children, barTintColor, selectedTintColor, unselectedTintColor, bottomTabs} = this.props;
+        var {children, barTintColor, selectedTintColor, unselectedTintColor, bottomTabs, scrollable} = this.props;
         var tabBarItems = React.Children.toArray(children).filter(child => !!child);
         var titleOnly = !tabBarItems.find(({props}: any) => props.title && typeof props.image === 'string');
         var tabLayout = Platform.OS === 'android' && (
             <NVTabLayout
                 selectedTintColor={selectedTintColor}
                 unselectedTintColor={unselectedTintColor}
                 selectedIndicatorAtTop={bottomTabs}
+                scrollable={scrollable?scrollable:false}
```

Comment:
Change to ```scrollable={scrollable}```. No need to check if populated because the default prop value handles this

Reply:
Ok